hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/MigrationRecordsManager.java

```diff
@@ -139,8 +146,14 @@ public void publishMigrationRecords(final Instant now) {
         publishAccountsCreated(
                 systemAccountsCreator.getSystemAccountsCreated(), now, SYSTEM_ACCOUNT_CREATION_MEMO, "system creation");
 
+        if (bootstrapProperties.getBooleanProperty(PropertyNames.ACCOUNTS_BLOCKLIST_ENABLED)) {
+            publishAccountsCreated(
+                    blocklistAccountCreator.getBlockedAccountsCreated(), now, BLOCKED_ACCOUNT_CREATION_MEMO, "blocked");
+        }
+
         curNetworkCtx.markMigrationRecordsStreamed();
         systemAccountsCreator.forgetCreations();
+        blocklistAccountCreator.forgetCreatedBlockedAccounts();
```

Comment:
what is the need for this if this is called one time only?  Or is it possible this method is called from somewhere _other_ than `ServicesTxnManager` above?